scripts/plot_p_nom_max.py

```diff
@@ -75,14 +62,15 @@ def cum_p_nom_max(net, tech, country=None):
 
     fig, axes = plt.subplots(1, len(techs))
 
-    for cluster in clusters:
-        net = pypsa.Network(getattr(snakemake.input, cluster))
+    for j, cluster in enumerate(clusters):
+        net = pypsa.Network(snakemake.input[j])
 
         for i, tech in enumerate(techs):
-            cum_p_nom_max(net, tech, country).plot(x="p_max_pu", y="c_p_nom_max", label=cluster, ax=axes[0][i], **plot_kwds)
+            cum_p_nom_max(net, tech, country).plot(x="p_max_pu", y="p_nom_max",
+                         label=cluster, ax=axes[i], **plot_kwds)
```

Comment:
It should be `axes[0][i]` shouldn't it? It's about the `i`th column.

Reply:
ah I see what you mean, but then we should include the subplot rows. before the lines of different cluster sizes (and one carrier) were plotted in the same subplot

Reply:
There's just a single row:
https://github.com/PyPSA/pypsa-eur/pull/107/files/1966cccec6824c86ca46095f230b32094ea1542d#diff-720aa7a680646ab82b6f89980ddb841aL76

Reply:
It was more or less the idea to compare how the maximum available capacity changes for different cluster sizes. For this it makes sense to have the different cluster sizes in the same panel and a different panel for each tech (completely different amnts). I don't have an example on my working computer unfortunately.

Reply:
https://github.com/PyPSA/pypsa-eur/issues/112#issuecomment-563300768
This is what comes out now, this is also what you mean or?

Reply:
> There's just a single row:
> https://github.com/PyPSA/pypsa-eur/pull/107/files/1966cccec6824c86ca46095f230b32094ea1542d#diff-720aa7a680646ab82b6f89980ddb841aL76

yes, it might be due to different pyplot version but for me axes[1] targets (in this case) at the first row and second column as axes is a one-dimensional list

Reply:
Ah okay, earlier there was a squeeze argument that defaulted to False maybe it's true now. I think its not cumulative in the plots.

Reply:
Did you add https://github.com/PyPSA/pypsa-eur/pull/107#discussion_r355507291 v?